File: JMovies.Entities/IMDB/Movie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace JMovies.Entities.IMDB
{
    public class Movie
    {
        public long IMDbID { get; set; }
        public string Title { get; set; }
        public string OriginalTitle { get; set; }
        public string PlotSummary { get; set; }
        public string StoryLine { get; set; }
        public Credit[] Credits { get; set; }
        public int Year { get; set; }
    }
}

File: JMovies.Web.App/src/app/models/general-models/movie.ts
import { ProductionTypeEnum } from "./production-type-enum";
import { Credit } from "./credit";
import { ActingCredit } from "./acting-credit";

export class Movie {
    imdbLink: string;
    imDbID: number;
    title: string;
    originalTitle: string;
    plotSummary: string;
    storyLine: string;
    credits: Credit | ActingCredit[];
    year: number;
    tagLines: string[];

    //public Keyword[] Keywords { get; set; }
    //public Genre[] Genres { get; set; }
    //public OfficialSite[] OfficialSites { get; set; }
    //public Country[] Countries { get; set; }
    //public Language[] Languages { get; set; }
    //public ReleaseDate[] ReleaseDates { get; set; }
    //public AKA[] AKAs { get; set; }
    filmingLocations: string[];
    //public Budget Budget { get; set; }
    //public Company[] ProductionCompanies { get; set; }
    //public TimeSpan Runtime { get; set; }
    productonType: ProductionTypeEnum;
}